prijateli_tree/app/controllers/games.py

```diff
@@ -0,0 +1,155 @@
+"""
+Contains baseline structure for the game.
+"""
+
+# Global imports
+import random
+from random import shuffle
+
+
+# Local imports
+# from prijateli_tree.app.controllers.players import Player
+
+
+class Game:
```

Comment:
I don't think this is the strategy go to use for the games or players. This strategy works if you can maintain state between accessing the API, but part of the REST philosophy is that it's stateless. You could certainly use classes, but each function NEEDS to be able to act without regard to any previous state. We cannot count on the connections between the server and a user being consistent generally, and especially since we may be working slower connections out in the field. The way that you'll maintain state for the API is by accessing the database itself.

You could use these classes, but they can really only serve a purpose through predominately static methods.